test/unit/Assistant.spec.ts

```diff
@@ -343,6 +344,40 @@ describe('Assistant class', () => {
           sinon.assert.calledWith(fakeClient.chat.postMessage, expectedParams);
         });
 
+        it('say should be called with message_metadata that supplements thread context', async () => {
```

Comment:
🙏 🚀

Reply:
@WilliamBergamin any word on when the next release will be cut?